tsconfig.json

```diff
@@ -1,12 +1,21 @@
 {
   "compilerOptions": {
-    "outDir": "./dist",
-    "allowJs": true,
-    "esModuleInterop": true,
     "strict": true,
-    "target": "es5",
-    "lib": ["es2015", "dom"]
+    "noUncheckedIndexedAccess": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
Why are we setting this to `true`? What are we skipping, here?

Reply:
Skipping any TS checks inside the `node_modules` folder